apps/dotcms-ui-e2e/src/pages/DotTemplates/DotTemplates.ts

```diff
@@ -0,0 +1,75 @@
+import Form from '../../support/shared/Form';
+import Navigation from '../../support/shared/Navigation';
+import Page from '../../support/shared/Page';
+
+const URL = '/dotAdmin/#/templates';
+const CREATE_TEMPLATE_BUTTON = 'dot-action-button dot-icon i';
+
+class DotTemplates {
+    static openPage() {
+        cy.intercept('GET', 'v1/templates').as('templates');
+        Navigation.visit(URL);
+        Navigation.assertPageUrlIs(URL);
+        cy.wait('@templates');
+    }
+
+    static checkTemplatesPageLoaded() {
+        Page.assertElementContains('.p-breadcrumb > ul > :nth-child(3)', `Templates`); // Header
+        Page.assertElementSize('p-table', 1); // Table
+        Page.assertElementSize('.action-header__global-search input[type="text"]', 1); // Global Search Input
+        Page.assertElementContains('.template-listing__header-options p-checkbox', `Show Archived`); // Show Archived button
+        Page.assertElementContains('.template-listing__header-options .p-button-label', `Actions`); // BulkAction button
```

Comment:
Should we add `data-testid` to all these?

Reply:
https://github.com/dotCMS/core-web/pull/1621/files#r593495180